.travis.yml

```diff
@@ -27,7 +27,7 @@ matrix:
       scala: 2.11.7
       env: TEST_SPARK_VERSION="1.5.0"
 script:
-  - sbt -Dspark.testVersion=$TEST_SPARK_VERSION ++$TRAVIS_SCALA_VERSION coverage test
```

Comment:
Even if Codecov isn't set up, this should still end up printing coverage metrics to the console, so there's no harm in leaving it in.